Client/Ecosia/Migration/EcosiaImport.swift

```diff
@@ -8,6 +8,10 @@ import MozillaAppServices
 import Storage
 import Shared
 
+private let tabsError = 911
```

Comment:
Wondering what is the reasoning behind the numbers, why starting in `911` for example?

Would it make sense to have instead an `enum` of errors with a rawValue of `Int`?

Reply:
:D yeah that's just arbitrary: like 911 call for ambulance.. I did not come up with a better idea and I don't know if there are dedicated error codes for failing migrations. Any suggestions? Also those are "only" temporary as well and can be removed after all users have migrated.. sure an enum makes sense -> will give that a shot

Reply:
I'm fine with starting on 911, just thought it had a meaning. I would use an enum and let it decide the numbers by default, but any solution works for me 👍